coral/src/app/features/topics/browse/hooks/environment/useGetEnvironments.test.tsx

```diff
@@ -90,6 +90,7 @@ describe("useGetEnvironments", () => {
           topicNameSuffix: undefined,
           defaultPartitions: undefined,
           defaultReplicationFactor: undefined,
+          type: "kafka",
```

Comment:
We need to send the environment `type` as part of the request payload for `getClusterInfo`, so the environment entity needed to be extended to include it.